### DISCUSSION TITLE
Stats over control or stats port

### DISCUSSION THREAD

**Discussion by TheSashmo** [CESNET/UltraGrid#159] [Q&A]
I am not sure if I missed it.  But I can control UG from a remote session using netcat, i.e nc 127.0.0.1 43552 seems simple enough.  But I would like to redirect the stats output to a central control.  I am not sure if I missed it, but I don't recall reading, or it wasn't in plain writing, on how to take the console output and re-direct to a log/database.  

Did I miss something?

**Comment by MartinPulec** [comment]
And what about using standard output redirection (or using [tee(1)](https://man7.org/linux/man-pages/man1/tee.1.html) if you need it to be also displayed at the same time)?

**Reply by TheSashmo** [reply]
Yeah I guess that can work, I could pump that back to nc in the other direction.

Thanks Martin.
